docs/contributor/contribute-docs.md

```diff
@@ -0,0 +1,184 @@
+---
+title: How to contribute docs
```

Comment:
@Tingtal Could you please take a look at this doc? you can get the preview at :
https://deploy-preview-145--karmada.netlify.app/docs/next/contributor/contribute-docs